avocado/plugins/spawners/podman.py

```diff
@@ -181,6 +181,19 @@ async def _create_container_for_task(self, runtime_task, env_args,
                             'avocado.core.nrunner',
                             'task-run']
 
+        test_opts = ()
+        if runtime_task. task.category == 'test':
```

Comment:
I'm assuming this whitespace was not intentional?  BTW, TIL: Python won't raise a `SyntaxError` on this.

Reply:
good catch, thanks. Yes, I am also surprised that I didn't get any `SyntaxError`